src/Item.hpp

```diff
@@ -57,7 +57,7 @@ class Item
 	, _path(folder)
 	, _sequence(sequence)
 	{
-		_path /= sequence.getStandardPattern();
+		_path /= sequence.getPrintfPattern();
```

Comment:
Because the returned name in case of a sequence with a variable padding, fails with the MikReader node at Mikros. This commit fixed this issue, and Nuke correctly interprets the new path with a printf style.